pkg/serverless/trace/trace_test.go

```diff
@@ -82,32 +82,70 @@ func TestLoadConfigShouldBeFast(t *testing.T) {
 }
 
 func TestFilterSpanFromLambdaLibraryOrRuntime(t *testing.T) {
-	spanFromLambdaLibrary := pb.Span{
+	httpSpanFromLambdaLibrary := pb.Span{
```

Comment:
nit: We could break these out into separate tests. Usually when writing unit tests, we want to focus on only asserting 1 behaviour per test. This let's us narrow down failures more easily. For instance, if the first assert fails, we won't know the results of the other tests.